src/components/TeacherComponents/ResourcePageSchoolDetails/ResourcePageSchoolDetails.tsx

```diff
@@ -74,7 +74,7 @@ const ResourcePageSchoolDetails: FC<ResourcePageSchoolDetailsProps> = ({
     setSchoolPickerInputValue(value);
   };
   return (
-    <OakBox $pa={"inner-padding-none"} $ba={"border-solid-none"} as="fieldset">
+    <OakFieldset $pa={"inner-padding-none"} $ba={"border-solid-none"}>
```

Comment:
minor, but the props `$pa={"inner-padding-none"} $ba={"border-solid-none"}` can be removed now I think.